web/src/lib/roadmap/roadmapBuilder.ts

```diff
@@ -9,7 +9,13 @@ export const buildRoadmap = async ({
   industryId: string | undefined;
   addOns: string[];
 }): Promise<Roadmap> => {
-  const stepImporter: () => Promise<GenericStep[]> = industryId ? importGenericSteps : importForeignSteps;
+  let stepImporter: () => Promise<GenericStep[]>;
+
+  if (industryId) {
+    stepImporter = industryId === "domestic-employer" ? importDomesticEmployerSteps : importGenericSteps;
```

Comment:
Note:  A suggestion for future work. 
To improve maintainability and reduce potential errors, I suggest we move string literals like "domestic-employer" and "test" into a dedicated config or constants file. This would make it easier to manage and update these values in the future, ensuring consistency across the codebase.

Reply:
Agreed.